tests/suites/test_suite_nist_kw.function

```diff
@@ -27,10 +27,6 @@ void mbedtls_nist_kw_self_test(int verbose, int mode, data_t *kw_key,
     psa_set_key_type(&attributes, PSA_KEY_TYPE_AES);
     mbedtls_printf("\n");
 
-    if (verbose != 0) {
-        mbedtls_printf("  KW-AES-%u ", (unsigned int) kw_key->len * 8);
-    }
-
```

Comment:
Thanks, but please also remove all other calls to `mbedtls_printf()` from this function, and the `verbose` parameter as well.